draft-ietf-taps-interface.md

```diff
@@ -191,12 +191,12 @@ itself; they are equivalent to Actions on a per-application global context.
 How these abstract concepts map into concrete implementations of this API in a
 given language on a given platform is largely dependent on the features of the
 language and the platform. Actions could be implemented as functions or method
-calls, for instance, and Events could be implemented via callbacks,
-communicating sequential processes, or other asynchronous calling conventions.
-The method for dispatching and handling Events is an implementation
-detail, with the caveat that the interface for receiving Messages must require
-the application to invoke the Connection.Receive() Action once per Message to be
-received (see {{receiving}}).
+calls, for instance, and Events could be implemented via event queues, handler
+functions or classes, communicating sequential processes, or other asynchronous
+calling conventions. The method for dispatching and handling Events is an
+implementation detail, with the caveat that the interface for receiving Messages
+must require the application to invoke the Connection.Receive() Action once per
+Message to be received (see {{receiving}}).
```

Comment:
This just got me confused. I wanted to complain about this saying "once per Message to be received" because I'm quite sure that there was a plan for @tfpauly to write that multiple message receives can be scheduled by just providing a parameter?  Yet, I don't see this in our current text and it's also not an open issue? Have we ended up deciding that it's really a 1-1 relationship after all?  Strange.....

Reply:
It would be nice to have this, maybe add it to the convenience features section?

Reply:
I don't know? What confuses me is that I'm quite sure that it was once agreed to change this, in general... I think @tfpauly even wanted it, and we said yes, and I even believe it happened???  Am I looking at the wrong section of the interface draft only, or what?

Reply:
Its definitely not in any of the drafts or Ive totally overread it as well. Having to call individual receives is one of the most annoying things about the PyTAPS API.

Reply:
@mwelzl maybe you are thinking about the similar discussion of a call to Listen returning more than one Connection? Anyway this seems to be a separate discussion.

Reply:
@mwelzl we agreed on this for listen, but not for recv. recv is a little different as it implicitly manages backpressure. Yust call recv in the event handler.

Reply:
Thanks, @abrunstrom and @philsbln for refreshing my memory!  Then I agree, it's a separate issue - but perhaps worth opening an issue for. I'll do that (and include a reminder that this particular text would have to be adapted along with the interface update in case we make this change).

Reply:
Sorry, I just changed my mind: I'm *not* opening an issue. This would complicate things (due to questions related to parameters), and I'm not going to add more complications. I might approve them at best  :-)   if someone else asks for it...